src/index.less

```diff
@@ -59,6 +59,24 @@ header .navbar.navbar-static-top {
     margin-left: 0.2em;
   }
 }
+
+.search-faceted {
+  margin-top: 6.5em;
+
+  & > ul {
+    padding: 0;
+  }
+  
+  ul.nav-stacked > li > a {
```

Comment:
I think using class as oppose to structure might scale better but it can be improved later,